index.ts

```diff
@@ -33,6 +34,7 @@ app.get('/', (req, res) => res.send('Express + TypeScript Server'));
 
 // ACCOUNT
 app.get('/accounts', listAccount);
+app.get('/accounts/:email', GetOneAccount);
```

Comment:
the word `one` is redundant here, we may call this function `getAccount` instead.